hibernate-cache/src/test/java/org/hibernate/test/cache/infinispan/stress/entities/Address.java

```diff
@@ -1,6 +1,7 @@
 <<<<<<< HEAD
 <<<<<<< HEAD
 <<<<<<< HEAD
+<<<<<<< HEAD
```

Comment:
Yeah, I realised I had not fix this stuff. I've created a new branch where I've done this properly and hence this is fixed there.